app/src/main/res/values/strings.xml

```diff
@@ -26,5 +26,8 @@
     <string-array name="detail_item_menu">
         <item>Pin to Dashboard</item>
     </string-array>
+
+    <string name="map_key">AIzaSyBQ8lY1t6WA0VsPZ5TMyKCU_UzmWjneark</string>
```

Comment:
Hi @UmarAuna welcome 🎉 
Nice findings, that's indeed should be oneliner of editText

but you should not commit this sensitive data to the master branch, and also, please try to commit only necessary changes to the PR. Thanks!

Reply:
Refactor: i have removed the sensitive information